CONTRIBUTING.md

```diff
@@ -82,7 +82,7 @@ use `make console` to load your application together with the framework located
 
 ```
 ghci
-:l IHP/exe/IHP/IDE/DevServer.hs
+:l IHP/ihp-ide/exe/IHP/IDE/DevServer.hs
```

Comment:
@mpscholten I think the doc is outdated.

1. Is `make console`, mentioned above , needed?
2. I'm getting an error
![image](https://github.com/user-attachments/assets/bdbe8084-ed40-4b01-bbec-2e54ae487f27)

Reply:
It could just be `ghci`. `make console` is just an alias for `ghci`.

Can you try `:set -iIHP/ihp-ide` in the ghci? This will add the `IHP/ihp-ide` directory to the load path. After that it should load without the error.

Reply:
Thanks, it helped (adapted docs)

But now I get

![image](https://github.com/user-attachments/assets/105a0cc2-aa99-4ad3-93c0-819ce7c07479)

Reply:
I can reproduce that. Just tried it locally and here's an approach how I got it working with latest IHP:

```
# Assuming workingDirectory is the project

cd IHP # switch to IHP
direnv allow # load direnv things if this is freshly cloned

ghci # Load ghci will all IHP dependencies
:cd .. # switch back to project directory
:set -iIHP
:set -iIHP/ihp-ide
 :l IHP/ihp-ide/exe/IHP/IDE/DevServer.hs
```

That should work now

locally i run into a ghc bug while doing that, I hope it works for you:

```
ghci> :l IHP/ihp-ide/exe/IHP/IDE/DevServer.hs
Ok, 164 modules loaded.
ghci> main
panic! (the 'impossible' happened)
  GHC version 9.8.2:
	nameModule
  internal $fHasQueryBuilderkJoinQueryBuilderWrapperjoinRegister1_iefl
  Call stack:
      CallStack (from HasCallStack):
        callStackDoc, called at compiler/GHC/Utils/Panic.hs:191:37 in ghc-9.8.2-inplace:GHC.Utils.Panic
        pprPanic, called at compiler/GHC/Types/Name.hs:341:3 in ghc-9.8.2-inplace:GHC.Types.Name
  CallStack (from HasCallStack):
    panic, called at compiler/GHC/Utils/Error.hs:503:29 in ghc-9.8.2-inplace:GHC.Utils.Error


Please report this as a GHC bug:  https://www.haskell.org/ghc/reportabug
```

Reply:
As a workaround for the GHC bug I reverted the commit that introduced the bug by running `git revert 4fe8b56e5a8c5dcb394a16b0dfb5eb2d6f77b18f` from within the `IHP` directory

Reply:
I've ran into more issues with the dev server right now and added a new helper called [`mainInParentDirectory` to fix that](https://github.com/digitallyinduced/ihp/commit/27a696af90efcd0f84260d84805dd2dc753e818c).

After you've synced this branch with latest master it should now work like this:
```
# Assuming workingDirectory is the project

cd IHP # switch to IHP
direnv allow # load direnv things if this is freshly cloned

ghci # Load ghci will all IHP dependencies
 :l ihp-ide/exe/IHP/IDE/DevServer.hs
mainInParentDirectory # Will run the IHP project in the parent project directory
```

Reply:
Thank you for looking into it :pray: 

I'm confused about which URL should be in my `flake.nix`

https://github.com/digitallyinduced/ihp/blob/27a696af90efcd0f84260d84805dd2dc753e818c/CONTRIBUTING.md?plain=1#L43-L67

I'm guessing we should always tell folks to point to the local IHP, or is there a different use case?